### ISSUE TITLE
Update cryptography in Docker image

### ISSUE THREAD

**Issue by calvinbui** [anchore/anchore-engine#733]
**Is this a request for help?**: No

---

**Is this a BUG REPORT or a FEATURE REQUEST?** (choose one): Feature Request

**Version of Anchore Engine and Anchore CLI if applicable**: 0.8.2

**What happened**: When I use Anchore to scan itself, the scan results reveal some issues for cryptography. I'm not sure if anyone else has brought this up or if it's even an issue:

My scan results:

```
docker.io/anchore/anchore-engine:v0.8.2
- GHSA-hggm-jpg3-v476: cryptography-3.1.1
```

**What did you expect to happen**: No issues.

**Any relevant log output from /var/log/anchore**:

**What docker images are you using**: The official ones from the helm charts.

**How to reproduce the issue**: Scan anchore-engine:0.8.2

**Anything else we need to know**:

**Comment by zhill** [comment]
Thanks @calvinbui we'll take a look. I think that's an indirect dependency but need to verify.

**Comment by zhill** [comment]
This is resolved in the 0.9.0-dev branch pending 0.9.0 release
